improver_tests/calibration/ensemble_calibration/test_EstimateCoefficientsForEnsembleCalibration.py

```diff
@@ -332,185 +253,160 @@ def test_with_kwargs(self):
-    def test_coefficients_from_mean_non_standard_units(self):
-        """Test that the expected coefficient cube is returned when the
-        historic forecast units are non-standard."""
-        expected_coeff_names = ["gamma", "delta", "alpha", "beta"]
-        self.historic_forecast.coord("forecast_period").convert_units("hours")
-        self.expected.coord("forecast_period").convert_units("hours")
-        result = self.plugin.create_coefficients_cube(
-            self.optimised_coeffs, self.historic_forecast
-        )
-        self.assertEqual(result, self.expected)
-        self.assertEqual(self.plugin.coeff_names, expected_coeff_names)
-
-    @ManageWarnings(ignored_messages=IGNORED_MESSAGES, warning_types=WARNING_TYPES)
-    def test_forecast_period_coordinate_not_present(self):
-        """Test that the coefficients cube is created correctly when the
-        forecast_period coordinate is not present within the input cube."""
-        self.expected.remove_coord("forecast_period")
-        self.expected.remove_coord("time")
-        self.historic_forecast.remove_coord("forecast_period")
-        result = self.plugin.create_coefficients_cube(
-            self.optimised_coeffs, self.historic_forecast
-        )
-        self.assertEqual(result, self.expected)
+        self.assertEqual(len(result), 4)
+        for cube in result:
+            self.assertEqual(
+                cube.coord("forecast_reference_time").cell(0).point, self.expected_frt,
+            )
+            self.assertArrayAlmostEqual(
+                cube.coord(axis="x").points, self.expected_x_coord_points
+            )
+            self.assertArrayAlmostEqual(
+                cube.coord(axis="x").bounds, self.expected_x_coord_bounds
+            )
+            self.assertArrayAlmostEqual(
+                cube.coord(axis="y").points, self.expected_y_coord_points
+            )
+            self.assertArrayAlmostEqual(
+                cube.coord(axis="y").bounds, self.expected_y_coord_bounds
+            )
+            self.assertDictEqual(cube.attributes, self.attributes)
 
-    @ManageWarnings(ignored_messages=IGNORED_MESSAGES, warning_types=WARNING_TYPES)
-    def test_model_configuration_not_present(self):
-        """Test that the coefficients cube is created correctly when a
-        model_configuration coordinate is not present within the input cube."""
-        self.expected.attributes.pop("mosg__model_configuration")
-        self.historic_forecast.attributes.pop("mosg__model_configuration")
-        result = self.plugin.create_coefficients_cube(
-            self.optimised_coeffs, self.historic_forecast
+        self.assertEqual([cube.name() for cube in result], self.expected_coeff_names)
+        self.assertArrayEqual(
+            result.extract("emos_coefficient_beta", strict=True)
+            .coord("realization")
+            .points,
+            self.historic_forecast_with_realizations.coord("realization").points,
```

Comment:
These are the bits of this test that are important to keep.  The metadata stuff is duplication.

Reply:
I've removed the duplicated assertions.